resources/js/core.js

```diff
@@ -255,12 +255,11 @@ const photoBooth = (function () {
             },
             success: (data) => {
                 console.log('picture processed', data);
-
-                if (data.error) {
```

Comment:
We still need this error check, because the error callback handles only request errors and this check checks for "expected" errors.

Reply:
Ok. Just curious where this error would come from in case no HTTP error is returned? I found nothing in `/api/applyEffects.php`? In case of any error `die()` gets executed and I assume that won't be a 200, would it? No problem to add this error handling again, I just would like to understand it. :-)

Reply:
If you pass a string to `die()` the message is returned and the script stops executing. Therefore we need the error check.

Reply:
Done.